app/models/review.rb

```diff
@@ -0,0 +1,8 @@
+class Review < ApplicationRecord
+  belongs_to :user
+  belongs_to :restaurant
+
+  validates :title, presence: true, length: { in: 3..30 }
+  validates :content, presence: true, length: { in: 5..300 }
+  validates :rating, presence: true
```

Comment:
thumbs up/thumbs down is 0 : 1 respectively?

Reply:
Ohh yeah. Either that, or 👎 = -1 while 👍 = 1?

Reply:
What should I change this to? validate only 0 and 1?

Reply:
👎 -> 0, 👍 ->  1 

in the future the rating range can be 1-5 (whole integers)